src/components/DealProposalForm.tsx

```diff
@@ -44,21 +50,30 @@ const FormInput = ({
 }) => {
   return (
     <div className="grid grid-cols-1 gap-4 mb-4">
-      <Field id="client-address" disabled={true} value={dealProposal.client.toString()}>
+      <Field
+        id="client-address"
+        disabled={true}
+        value={dealProposal.client.toString()}
+        tooltip="Your blockchain address that will be associated with this storage deal"
+      >
         Client Address
       </Field>
 
       {/* TODO: add CID validation */}
-      <Field id="piece-cid" value={dealProposal.pieceCid.toString()}>
+      <Field
+        id="piece-cid"
+        value={dealProposal.pieceCid.toString()}
+        tooltip="Content Identifier - a unique hash that identifies your data"
+      >
         Piece CID
       </Field>
 
       <Field
         id="piece-size"
         type="number"
         value={dealProposal.pieceSize}
+        tooltip="Piece size in bytes"
```

Comment:
```suggestion
        tooltip="Piece size after the raw data was pre-processed and put into a CARv2 file. It can by generated by `polka-storage-provider-client proofs commp <carv2>."
```